packages/flutter_tools/test/general.shard/crash_reporting_test.dart

```diff
@@ -100,7 +100,7 @@ void main() {
     expect(logger.statusText, contains('NoPIIFakeDoctorText'));
     expect(logger.statusText, isNot(contains('Ignored')));
     expect(logger.statusText, contains('https://github.com/flutter/flutter/issues/new'));
-    expect(logger.errorText, contains('A crash report has been written to ${file.path}.'));
+    expect(logger.errorText, contains('A crash report has been written to ${file.path}'));
```

Comment:
This test change actually won't verify this change, as this will still pass if there's a period at the end.

Reply:
I reverted my commit & kept the necessary change!

Thank you for your review! 

I am new to the open-source! Started this `good-first-issue` but facing a lot of test cases failed in my initial commit only 😢. I am figuring it out!

Reply:
> I reverted my commit & kept the necessary change!
> 
> Thank you for your review!
> 
> I am new to the open-source! Started this `good-first-issue` but facing a lot of test cases failed in my initial commit only 😢. I am figuring it out!

Don't worry about it. Flutter is a huge project, and it's not surprising for obscure tests to break from the simplest changes 🙂